code/game/objects/structures/lamarr_cage.dm

```diff
@@ -94,4 +94,39 @@
 	setGender(FEMALE)
 
 /obj/item/clothing/mask/facehugger/lamarr/New()//to prevent deleting it if aliums are disabled
-	return
\ No newline at end of file
+	create_reagents(15)
+	return
+
+/obj/item/clothing/mask/facehugger/lamarr/process()
+	if(!istype(loc, /mob/living/carbon/human))
+		processing_objects.Remove(src)
+		return
+	var/mob/living/carbon/human/H = loc
+	if(src.reagents)
+		for (var/datum/reagent/current_reagent in src.reagents.reagent_list)
+			if (current_reagent.id == "creatine")
+				to_chat(H, "<span class='warning'>[src]'s body contorts and expands!</span>")
+				var/hand = null
```

Comment:
I don't see the point of a variable like this since it's only used in the `if(hand == "r")` check, why not do the `if(H.r_hand == src)` check there instead?